12LessonTwelve/in-class-project/css/style.css

```diff
@@ -1 +1,32 @@
+body {
+  margin: 0;
+  padding: 0;
+}
 
+.banner {
+  margin:0;
+  background-image: url(../img/sub_header_home.jpg);
+  height:70vh;

```

Comment:
remember to make sure that your classes don't interfere with bootstrap's own. a container-fluid here would've taken care of your width issue. replace those containers with container-fluids please!